root/static/scripts/common/utility/sortByEntityName.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * @flow strict
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import sortBy from 'lodash/sortBy';
+
+export default function sortByEntityName(
+  entities: $ReadOnlyArray<CoreEntityT>,
+): $ReadOnlyArray<CoreEntityT> {
+  return sortBy(entities, [entity => entity.name.toLowerCase(), 'id']);
```

Comment:
This only works with ASCII characters in `en` locale.
I would expect sort to follow user selected locale instead.

Reply:
Changed it to use i18n/compare, which does sort é with e as expected :)